tests/suites/0_stateless/18_rbac/18_0006_mysql_auth.py

```diff
@@ -35,6 +36,9 @@
     cursor.execute(
         "create user u3 identified by 'abc123' with set network policy='p2';"
     )
+    cursor.execute(
+        "create user u4 identified by 'abc123' with must_change_password = true;"
```

Comment:
Better to mock a password failed test? e.g.

create user u4 identified by 'abc123' with must_change_password = false;

Bound a password policy that max_failed_connect 3 times to the user u5, after failed 3 time, the u5 can be login with correct password. But only support modify password?

Reply:
the default value of `must_change_password` is false, `u1`, `u2` and `u3` is this case, we don't need add a new user.
the min value of `max_age_days` in password policy is 1 day, it means we must wait 1 day to test this case, it not easy to test here.